daemon/execdriver/windows/run.go

```diff
@@ -228,14 +172,23 @@ func (d *Driver) Run(c *execdriver.Command, pipes *execdriver.Pipes, startCallba
 	logrus.Debugf("CommandLine: %s", createProcessParms.CommandLine)
 
 	// Start the command running in the container.
-	var pid uint32
-	pid, err = hcsshim.CreateProcessInComputeSystem(c.ID, createProcessParms)
-
+	pid, stdin, stdout, stderr, err := hcsshim.CreateProcessInComputeSystem(c.ID, pipes.Stdin != nil, true, !c.ProcessConfig.Tty, createProcessParms)
```

Comment:
We looked long and hard at that. In some ways, I would say yes, but there are subtle differences between them. Our end conclusion was that it was cleaner (and in far easier to read/understand/maintain the code without introducing subtle bugs) being separate.